File: Creational/Builder/Python/ConcreteProduct1.py
from AbstractProduct import *

class ConcreteProduct1(AbstractProduct):

	def step_a_1(self):
		self.string_set.add("step_a_1")
		
	def step_a_2(self):
		pass
	
	def step_b_1(self):
		self.string_set.add("step_b_1")
	
	def step_b_2(self):
		pass
	
	def step_c_1(self):
		self.string_set.add("step_c_1")
	
	def step_c_2(self):
		pass

File: Creational/Builder/Java/ConcreteProduct1.java

public class ConcreteProduct1 extends AbstractProduct{

	@Override
	public void stepA1() {
		this.string_set.add("stepA1");
	}

	@Override
	public void stepA2() {
	}

	@Override
	public void stepB1() {
		this.string_set.add("stepB1");
	}

	@Override
	public void stepB2() {
	}

	@Override
	public void stepC1() {
		this.string_set.add("stepC1");
	}

	@Override
	public void stepC2() {
	}

}
